admin_guide/configuring_authentication.adoc

```diff
@@ -40,32 +49,38 @@ configuration file]. The following sections detail the identity providers
 supported by OpenShift.
 
 There are three parameters common to all identity providers:
+
 [cols="2a,8a",options="header"]
 |===
 |Parameter     | Description
-.^|`name`      | The provider name is prefixed to provider usernames to form an identity name.
-.^|`challenge` | When `true`, unauthenticated token requests from non-web clients (like
-the CLI) are sent a `WWW-Authenticate` challenge header. Not supported by all
-identity providers.
-
-NOTE: To prevent CSRF attacks against browser clients, Basic authentication challenges
-are only sent if a `X-CSRF-Token` header is present on the request. Clients that expect
-to receive Basic `WWW-Authenticate` challenges should set this header to a non-empty value.
-
-.^|`login`     | When `true`, unauthenticated token requests from web clients (like the web
-console) are redirected to a login page backed by this provider. Not supported
-by all identity providers.
+.^|`name`      | The provider name is prefixed to provider user names to form an
+identity name.
+.^|`challenge` | When `true`, unauthenticated token requests from non-web
+clients (like the CLI) are sent a `WWW-Authenticate` challenge header. Not
+supported by all identity providers.
+
+[NOTE]
+====
+To prevent CSRF attacks against browser clients, Basic authentication
+challenges are only sent if a `X-CSRF-Token` header is present on the request.
+Clients that expect to receive Basic `WWW-Authenticate` challenges should set
+this header to a non-empty value.
+====
+
+.^|`login`     | When `true`, unauthenticated token requests from web clients
+(like the web console) are redirected to a login page backed by this provider.
+Not supported by all identity providers.
 |===
 
 [[AllowAllPasswordIdentityProvider]]
 
 === Allow All
-Set `*AllowAllPasswordIdentityProvider*` in the `*identityProviders*` stanza to
+Set *AllowAllPasswordIdentityProvider* in the `*identityProviders*` stanza to
```

Comment:
I left that markup for identityProviders since a "stanza" seemed to fit best as a "config param / env var": https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference